src/NOTD.SC2Map/Base.SC2Data/GameData/AbilData.xml

```diff
@@ -3154,6 +3154,7 @@
         <Range value="5"/>
         <AutoCastRange value="5"/>
         <TargetFilters value="Heroic;Neutral,Enemy"/>
+        <AutoCastValidatorArray value="PlayerHas5000XP"/>
```

Comment:
The shop will be open to the target player (shop user) if the validator is true.